matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/SyncResponsePostTreatmentAggregatorHandler.kt

```diff
@@ -68,4 +76,21 @@ internal class SyncResponsePostTreatmentAggregatorHandler @Inject constructor(
             updateUserAccountDataTask.execute(UpdateUserAccountDataTask.DirectChatParams(directMessages = directChats))
         }
     }
+
+    private suspend fun fetchAndUpdateUsers(usersToFetch: List<String>) {
+        val userProfiles = fetchUsers(usersToFetch)
+        userProfiles.forEach { saveUserLocally(monarchy, it) }
+    }
+
+    private suspend fun fetchUsers(usersToFetch: List<String>) = usersToFetch.map {
+        val profileJson = profileService.getProfile(it)
+        User.fromJson(it, profileJson)
+    }
+
+    private fun saveUserLocally(monarchy: Monarchy, user: User) {
```

Comment:
monarchy is a class member, there is no need to add it as a method parameter.

Reply:
istg i'm actually blind